### PR TITLE
:whale: :building_construction: feat(docker): laradock configuration …

### PR DESCRIPTION
**Pull request by Sylcab74**
…and creating makefile

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,23 @@
+workspace_container=mentalist_workspace_1
+
+install:
+	docker exec $(workspace_container) composer install; reset; npm install; reset
```
Comment (JrBour) [inline Makefile]: Can we use yarn instead of npm ?

Comment (JrBour) [reply Makefile]: If it takes time, forget it

Comment (Sylcab74) [reply Makefile]: What is `yarn` ?

**Comment by JrBour** [comment]
The command `make start` doesn't works, because the folder `laradock` is empty.

**Comment by Sylcab74** [comment]
to pull git submodules : `git pull --recurse-submodules`
And ask me for the laradock `.env`

**Pull request merged**